### PR TITLE
xfconf: fix dbus may not be started in the startup of NixOS.

### PR DESCRIPTION
**Pull request by gou4shi1**
### Description

fix https://github.com/nix-community/home-manager/pull/3378#issuecomment-1368015573
ref: https://github.com/nix-community/home-manager/blob/693d76eeb84124cc3110793ff127aeab3832f95c/modules/misc/dconf.nix#L102-L106
output:
```
Feb 26 23:58:53 laptop hm-activate-cgq[1076]: Activating xfconfSettings
Feb 26 23:58:53 laptop hm-activate-cgq[1455]: dbus-daemon[1455]: [session uid=1000 pid=1455] Activating service name='org.xfce.Xfconf' requested by ':1.0' (uid=1000 pid=1458 com>
Feb 26 23:58:53 laptop hm-activate-cgq[1455]: dbus-daemon[1455]: [session uid=1000 pid=1455] Successfully activated service 'org.xfce.Xfconf'
Feb 26 23:58:53 laptop xfconfd[1465]: Name org.xfce.Xfconf lost on the message dbus, exiting.
Feb 26 23:58:53 laptop systemd[1]: Finished Home Manager environment for cgq.
```

### Checklist

- [x] Change is backwards compatible.

- [x] Code formatted with `./format`.

- [ ] Code tested through `nix-shell --pure tests -A run.all`.

- [ ] Test cases updated/added. See [example](https://github.com/nix-community/home-manager/commit/f3fbb50b68df20da47f9b0def5607857fcc0d021#diff-b61a6d542f9036550ba9c401c80f00ef).

- [x] Commit messages are formatted like

    ```
    {component}: {description}

    {long description}
    ```

    See [CONTRIBUTING](https://github.com/nix-community/home-manager/blob/master/docs/contributing.adoc#sec-commit-style) for more information and [recent commit messages](https://github.com/nix-community/home-manager/commits/master) for examples.

- If this PR adds a new module

  - [ ] Added myself as module maintainer. See [example](https://github.com/nix-community/home-manager/blob/068ff76a10e95820f886ac46957edcff4e44621d/modules/programs/lesspipe.nix#L6).

  - [ ] Added myself and the module files to `.github/CODEOWNERS`.

### CODE REVIEW

File: modules/misc/xfconf.nix
Comment (ncfavier) [inline modules/misc/xfconf.nix:105]:
```suggestion
          export XFCONF_DBUS_RUN_SESSION=""
```

**Review by ncfavier — CHANGES_REQUESTED** [review]
This seems to be taken verbatim from the dconf module; some deduplication would be nice, though I don't have any good ideas that don't force a dependency on `dbus`.

Please squash the commits into one and name it according to the [contribution guidelines](https://github.com/nix-community/home-manager/blob/master/docs/contributing.adoc#sec-commit-style).

**Comment by ncfavier** [comment]
Oh, and please target `master`. We'll backport later if needed.

**Comment by gou4shi1** [comment]
@ncfavier Thanks for your review :)

**Review by ncfavier — APPROVED** [review]

**Comment by gou4shi1** [comment]
Would it be merged?

**Comment by ncfavier** [comment]
I'm giving the other maintainers some time to review this; will merge in a couple days if no one objects.

**Pull request merged**
